README.md

```diff
@@ -1,99 +1,60 @@
-reviews, and you can find the Poseidon review board there:
-
-https://review.gerrithub.io/#/q/project:camsas/poseidon+is:open
-
-In order to do code reviews, you will need an account on GerritHub (you can link
-your GitHub account).
-
-The easiest way to submit changes for review is to check out Poseidon from
-GerritHub, or to add GerritHub as a remote. Alternatively, you can submit a pull
-request on GitHub and we will import it for review on GerritHub.
+# Introduction
+The Poseidon/Firmament scheduler incubation project is to bring integration of Firmament Scheduler [OSDI paper](https://www.usenix.org/conference/osdi16/technical-sessions/presentation/gog) in Kubernetes.
+At a very high level, Poseidon/Firmament scheduler augments the 
+current Kubernetes scheduling capabilities by incorporating a new 
+novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. 
+Firmament models workloads on a cluster as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
+
+Due to the inherent rescheduling capabilities, the new scheduler enables a globally optimal scheduling for a given policy that keeps on refining the dynamic placements of the workload.
+
+As we all know that as part of the Kubernetes multiple schedulers support, each new pod is typically scheduled by the default scheduler, but Kubernetes can be instructed to use another scheduler by specifying the name of another custom scheduler (Poseidon in our case) at the time of pod deployment. In this case, the default scheduler will ignore that Pod and allow Poseidon scheduler to schedule the Pod on a relevant node. We plugin Poseidon as an add-on scheduler to K8s, by using the 'schedulerName' as Poseidon in the pod template this will by-pass the default-scheduler. We plugin Poseidon as an add-on scheduler to K8s, by using the 'schedulerName' as Poseidon in the pod template this will by-pass the default-scheduler.
+
+# Key Advantages
+
+* Flow graph scheduling provides the following 
+  * Support for high-volume workloads placement.
+  * Complex rule constraints. 
+  * Globally optimal scheduling for a given policy.
+  * Extremely high scalability. 
+  
+  **NOTE:** Additionally, it is also very important to highlight that Firmament scales much better than default scheduler as the number of nodes increase in a cluster.
+
+# Design 
+
+   <p align="center">
+  <img src="docs/poseidon.png"> 
+<p align="center"> <b>Poseidon Integration architecture</b> </p>
+</p>
+
+
+
+For more details about the design of this project see the [design document](https://docs.google.com/document/d/1VNoaw1GoRK-yop_Oqzn7wZhxMxvN3pdNjuaICjXLarA/edit?usp=sharing) doc.
+
+
+
+# Installation
+  In-cluster installation of Poseidon, this method is to run Poseidon and its components in the
+  kubernetes cluster.
+  
+  The following are the components required.
+   * A [Running Kubernetes cluster](https://kubernetes.io/docs/setup/).
```

Comment:
What is A doing here in the text? Did you mean "Running a Kubernetes cluster" instead?